ENGAGEHF/Managers/NotificationManager/NotificationManager.swift

```diff
@@ -119,4 +172,20 @@ class NotificationManager: Module, NotificationHandler, NotificationTokenHandler
         
         self.logger.debug("Successfully registered device for remote notifications.")
     }
+    
+    
+    private func unregisterDeviceToken(_ token: Data?) async throws {
```

Comment:
Wondering if we can
1. Take the token from the locally stored device token instead of a function argument
2. Move the setting to nil of the device token in there
3. Even not store the token at all and just get it from the Notifications Module using the register function? Is there a way to check if we get a new token very time or if it is stable? If it is stable, we could just call the method every time. We can probably easily check this with a small loop and print statements.